README.md

```diff
@@ -0,0 +1,162 @@
+
+### ShoppingCartController
+
+For a personalized shopping experience, the ShoppingCartController takes care of user shopping carts and the individual cart items within our Online Book Store API. Users can create and manage their shopping carts, add and remove items, and review their cart contents effortlessly through this controller. It ensures the smooth functioning of shopping cart-related operations, allowing users to curate their selections before proceeding to checkout. This controller enhances the user's ability to build and manage their reading lists.
+
+### OrderController
+
+The OrderController is the hub for managing user orders and the real-time status updates associated with them in our Online Book Store. Users can place orders, review their order history, and monitor order statuses seamlessly through this controller. It facilitates order creation, retrieval, modification, and deletion, ensuring a smooth and reliable ordering process. By centralizing order management, this controller enhances user confidence in our e-commerce platform and offers transparency in the order fulfillment process.
+
+## Getting Started
+
+To set up and use the Online Book Store API, follow these steps:
+
+- Clone this repository to your local machine.
+- Configure the application properties and database settings in `application.properties`.
+- Building and Running the API
+
+#### Using Docker (Recommended)
+
+1. Build the Docker image using the provided Dockerfile:
+
+   ```bash
+   docker-compose build
+   ```
+
+2. Run the container with the built image:
+
+   ```bash
+   docker-compose up
+   ```
+
+   Use the `-p` flag to specify the port on which your application will be available.
+
+3. Your API is now accessible at [http://localhost:8080](http://localhost:8080).
+
+#### Without Docker
+
+1. Build the project using Maven:
+
+   ```bash
+   mvn clean install
+   ```
+
+2. Run the application:
+
+   ```bash
+   mvn spring-boot:run
+   ```
+
+   The API will be available at [http://localhost:8080](http://localhost:8080).
+
+
+- Access the API documentation and testing interface by visiting 
+[http://localhost:8080/swagger-ui.html](http://localhost:8080/swagger-ui.html)
+- Start making API requests as per the documentation. You can use [Online Book Store.postman_collection.json](Online%20Book%20Store.postman_collection.json)
+- Here is a [video](https://www.loom.com/share/0eed450468064439ae5a1cd9a741165d?sid=f9157ee2-5b5f-445b-b17f-56a5e6fc06f7) where we can see our API in action 
```

Comment:
I think it will be better to speak in the video for a better understanding of actions performed